recipes/taxoniq/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Can you bring the fact that this file isn't included up upstream?

They can "fix" this by adding a metadata section:
https://github.com/scikit-image/scikit-image/blob/main/setup.cfg#L10

Reply:
What do you mean by bringing it up?

Reply:
Open an issue and let them know that their license file is not included in their tar.gz file from:
https://pypi.org/project/taxoniq/#files

they probably want it to be.

Reply:
Thanks, I opened the issue.